packages/node-experimental/src/integrations/getAutoPerformanceIntegrations.ts

```diff
@@ -0,0 +1,89 @@
+import type { Integration, IntegrationClass } from '@sentry/types';
+import { dynamicRequire } from '@sentry/utils';
+
+import type { Express } from './express';
+import type { Fastify } from './fastify';
+import type { GraphQL } from './graphql';
+import type { Mongo } from './mongo';
+import type { Mongoose } from './mongoose';
+import type { Mysql } from './mysql';
+import type { Mysql2 } from './mysql2';
+import type { Nest } from './nest';
+import type { Postgres } from './postgres';
+import type { Prisma } from './prisma';
+
+const INTEGRATIONS = [
```

Comment:
l (no action required): 
I wonder if this is a good idea. The requires prolong startup time and that's an issue for serverless environments. But as I said, no action required at this time, we can revisit this later.

Reply:
You mean to use this by default? The way this works I copied from the node auto discovery, only there it is opt-in via `Sentry.autoDiscoverNodePerformanceMonitoringIntegrations()`.

With this setup, if you want to/need to avoid this, you can disable default integrations and opt-in to just the ones you want. Not sure if that is the best approach 🤔

Reply:
Yes, I was concerned about making this opt-in. iirc we discussed dropping `autoDiscoverNodePerformanceMonitoringIntegrations` entirely in v7 because of the performance implications but decided to make it opt-in. I don't want to block this PR though so it's fine as is for now. Just wanted to bring this up.

Reply:
Yeah, we can def. re-evaluate this, for now this is really experimental, and this specific part is easily changed later :) But let's keep this in mind! 👍